ptf/platform_helper/bfn_sai_helper.py

```diff
@@ -0,0 +1,20 @@
+import sys
+# from common_sai_helper import CommonSaiHelper
+sys.path.append("..")
```

Comment:
refers to [bfn/brcm/mlnx]_sai_helper: 
1. will we need this import or should we remove it?
2. what is the purpose of path appending?

Reply:
Great POINTS!
optimized